### PR TITLE
fix that in UGV dont work speedLimiter

### LINKED ISSUE #2875: Can't use speed limiter on UGV
ACE3 Version: 3.3.2

**Mods:**
* `@CBA_A3`
* `@ace`

**Placed ACE3 Modules:**
* *None*

**Description:**
* It's not possible to enable speed limiter while driving UGV.*

**Steps to reproduce:**
* *Control UGV and try to enable speed limiter.*

**Where did the issue occur?**
*"Multiplayer", "Editor" or "Singleplayer"*

**RPT log file:**
* -

### PR DESCRIPTION
**Pull request by jokoho48**
posible fix #2875

### CODE REVIEW

File: addons/vehicles/XEH_postInit.sqf [hunk 1]
```diff
@@ -2,20 +2,31 @@
...
-    // Conditions: canInteract
-    if !([ACE_player, objNull, ["isnotinside"]] call EFUNC(common,canInteractWith)) exitWith {false};
-    // Conditions: specific
-    if !(ACE_player == driver vehicle ACE_player &&
-    {vehicle ACE_player isKindOf 'Car' ||
-        {vehicle ACE_player isKindOf 'Tank'}}) exitWith {false};
+    private _connectedUAV = getConnectedUAV ACE_player;
+    private _uavControll = UAVControl _connectedUAV;
+    if ((_uavControll select 1) == "DRIVER") then {
+        if !(_connectedUAV isKindOf "UGV_01_base_F") exitWith {false};
```
Comment (commy2) [inline addons/vehicles/XEH_postInit.sqf]: No!!!

Comment (jonpas) [inline addons/vehicles/XEH_postInit.sqf]: Then we need a function in common, eg. `EFUNC(common,isUAV)`.

Comment (jokoho48) [inline addons/vehicles/XEH_postInit.sqf]: this case only need UGVs and not support UAVs

Comment (PabstMirror) [inline addons/vehicles/XEH_postInit.sqf]: `isUav = 1;`

Not many UAV's that are also a car/tank; so hard coded isn't terrible (this is a pretty odd edge case)

**Comment by gienkov** [comment]
:+1:

**Pull request merged**